CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 3.2.0 (2020-01-06)
+## 3.3.0 (2020-03-24)
+
+- Bump acorn from 6.3.0 to 6.4.1 to address security vulnerability. [#56](https://github.com/blackbaud/skyux-lib-help/pull/56)
+- Add capability for extending configuration at runtime. [#57](https://github.com/blackbaud/skyux-lib-help/pull/57)
```

Comment:
I'd change `Add` to `Added` to follow our pattern. Also, add the missing article `a` before `capability` and the missing article `the` before `configuration`. And I'd change `for extending` to `to extend` to avoid the gerund.

Again, I'll go ahead and make these changes, but I'm noting them here in case we need to reconsider any of the changes.